src/plugins/dam/utils/damUrl.js

```diff
@@ -10,7 +10,7 @@ const tld = API_ENDPOINT.split('://').pop().split('/').shift()
 
 export const baseUrls = {
   dfault: `https://dam.${APP_ENV === 'prod' ? '' : `${APP_ENV}.`}${tld}/`,
-  image: `https://images${APP_ENV === 'prod' ? '.' : `-${APP_ENV}.`}${tld}/`,
+  image: `https://imagez${APP_ENV === 'prod' ? '.' : `-${APP_ENV}.`}${tld}/`,
```

Comment:
@gdbrown due to the differences in the image base urls i.e. https://imagez-dev.toofab.com vs. https://bachelornation-images-dev.akamaized.net. Can we just set something like: IMAGE_BASE_URL=https://imagez-dev.toofab.com/ in the env file and use image:IMAGE_BASE_URL here?

Reply:
@albertgcsula not opposed to that, was trying to get them to be the same strategy as the symfony apps and let the index.js or config on the local site to customize it as needed.  either way.  if using globals then you have some more wiring to do in other places and you'd want to do DAM_BASE_URL and IMAGE_BASE_URL